src/software/ai/hl/stp/tactic/test_tactics/stop_test_tactic.cpp

```diff
@@ -15,11 +15,11 @@ double StopTestTactic::calculateRobotCost(const Robot &robot, const World &world
     return 0.5;
 }
 
-void StopTestTactic::calculateNextIntent(IntentCoroutine::push_type &yield)
+void StopTestTactic::calculateNextAction(ActionCoroutine::push_type &yield)
 {
     do
     {
-        yield(std::make_unique<StopIntent>(this->robot->id(), false, 0));
+        yield(std::make_shared<StopAction>());
```

Comment:
This should yield a proper StopIntent?

Reply:
Similar comment here to the one in `move_test_tactic.cpp`.